### PR TITLE
Fix slave_info miss label values

### PR DESCRIPTION
**Pull request by yz1509**

### CODE REVIEW

**Comment by oliver006** [comment]
Good catch, thanks for the PR!

Can you please add tests around this scenario so this won't happen again in the future?

**Comment by oliver006** [comment]
Adding @jtorkkel to the conversation to see if he has suggestions as to what a test could look like,  he added the master_host|port labels in the first place.

File: exporter_test.go [hunk 1]
```diff
@@ -1325,6 +1325,14 @@ func TestClusterSlave(t *testing.T) {
 			t.Errorf("Did not find key [%s] \nbody: %s", want, body)
 		}
 	}
+	for _, unwanted := range []string{
```
Comment (oliver006) [inline exporter_test.go:1328]: Can we invert this test and check for presence of expected values?

Comment (jtorkkel) [reply exporter_test.go:1328]: Hi, did not fully understood the question. In order to create and run multimaster tests you need to have KeyDB installed or you needs to include mock "info replication" for multimaster test cases. Redis OSS does not support multimaster, only simgle master nad multiple replicas. Redis EE, support active-active which is similar but not sure if same fields exposed through info.

The following info replication mock could be used for multimaster. Master 0 and 1 are up, Master 2 down.
127.0.0.1:8881> info replication
# Replication
role:active-replica
Master 0:
master_host:127.0.0.1
master_port:8882
master_link_status:up
master_last_io_seconds_ago:6
master_sync_in_progress:0
slave_repl_offset:16291
Master 1:
master_host:127.0.0.1
master_port:8883
master_link_status:up
master_last_io_seconds_ago:6
master_sync_in_progress:0
slave_repl_offset:16513
Master 2:
master_host:127.0.0.1
master_port:8884
master_link_status:down
master_last_io_seconds_ago:-1
master_sync_in_progress:0
slave_repl_offset:12740
master_link_down_since_seconds:62
slave_priority:100
slave_read_only:0
connected_slaves:2
slave0:ip=127.0.0.1,port=8882,state=online,offset=24123,lag=1
slave1:ip=127.0.0.1,port=8883,state=online,offset=24123,lag=1
master_replid:b11b52dc9c93567be5c05b1bee23d2c73c694457
master_replid2:0000000000000000000000000000000000000000
master_repl_offset:24123
second_repl_offset:-1
repl_backlog_active:1
repl_backlog_size:1048576
repl_backlog_first_byte_offset:1
repl_backlog_histlen:24123
127.0.0.1:8881>

Comment (yz1509) [reply exporter_test.go:1328]: In the current implementation, `slave_info["master_host"]` and `slave_info["master_port"]` are always empty.

Comment (jtorkkel) [reply exporter_test.go:1328]: Seem that same variable used in slave showing master address and in multimaster other masters. 
Removing "continue" seems to fix the issue based on my testing and not breaking multimaster.


OSS slave:
127.0.0.1:8882> info replication
# Replication
role:slave
master_host:127.0.0.1
master_port:8881
master_link_status:up
master_last_io_seconds_ago:8
master_sync_in_progress:0
slave_repl_offset:308
slave_priority:100
slave_read_only:1
connected_slaves:0
master_port:8881

File: exporter_test.go [hunk 2]
```diff
@@ -1325,6 +1325,14 @@ func TestClusterSlave(t *testing.T) {
 			t.Errorf("Did not find key [%s] \nbody: %s", want, body)
 		}
 	}
+	for wantedKey, wantedVal := range map[string]int{
+		`master_host="192.168.48.8"`: 5,
```
Comment (oliver006) [inline exporter_test.go:1329]: Can you shorten this to just `192.168.`? I _think_ there's no guarantee the container will always get the same IP but the subnet should be consistent.

Comment (yz1509) [reply exporter_test.go:1329]: port changes too 2333

Comment (oliver006) [reply exporter_test.go:1329]: Isn't the port fixed?

Comment (yz1509) [reply exporter_test.go:1329]: Last time it was 7002, this time it was 7001, is the prefix 700?

Comment (oliver006) [reply exporter_test.go:1329]: Ah, you're right, I guess the three hosts select a master and yeah, they ports all start with `700` --> https://github.com/oliver006/redis_exporter/blob/master/.drone.yml#L54

Comment (yz1509) [reply exporter_test.go]: `master_host="172.20.0.9"`  = =

Comment (oliver006) [reply exporter_test.go]: `172.20.` is fine, too. That'll be the IP range for docker in drone.

File: exporter_test.go [hunk 3]
```diff
@@ -1325,9 +1326,13 @@ func TestClusterSlave(t *testing.T) {
 			t.Errorf("Did not find key [%s] \nbody: %s", want, body)
 		}
 	}
+	hostReg, _ := regexp.Compile(`master_host="([0,1]?\d{1,2}|2([0-4][0-9]|5[0-5]))(\.([0,1]?\d{1,2}|2([0-4][0-9]|5[0-5]))){3}"`)
```
Comment (yz1509) [inline exporter_test.go:1329]: regular expressions seem to work

Comment (oliver006) [reply exporter_test.go:1329]: 👍

**Comment by oliver006** [comment]
Thanks!
